virttest/qemu_vm.py

```diff
@@ -49,6 +49,10 @@
 from virttest.qemu_devices.utils import DeviceError
 from virttest.qemu_capabilities import Flags
 
+# Overriding a module here and using as lower capital is not the best way to
+# do, but this is just a workaround to avoid changing the entire file.
+logging = logging.getLogger('avocado.' + __name__)
```

Comment:
I have done similar fixes on the side of our plugin since changing each line is both much work and very error prone. IN this case I would recommend keeping the logging module accessible via something like
```python
import logging as log
logging = log.getLogger('avocado.' + __name__)
```
This is probably just a minor draft so it is good enough for testing of course.

Reply:
Fixed with a force-push. Thanks.